styles.css

```diff
@@ -0,0 +1,328 @@
+
+/*//////////////////////////
+Table of Contents
+ 1. Typografie
+ 2. Global styles
+ 3. Layout
+ 3.1 Reusables
+ 3.2 Others
+ 4. Areas
+ 4.1 Header
+ 4.2 Main
+ 4.3 Footer
+ 5. Media Queries
+ ///////////////////////////*/
+
+/*/////////////////////////////////////////
+        1.Typografie
+////////////////////////////////////////// */
+
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@700&family=Roboto&display=swap');
+
+
+/*/////////////////////////////////////////
+        2.Global Styles
+////////////////////////////////////////// */
+/* Hierin wordt opgenomen: CSS reset en
+ Variabelen */
+
+/*CSS reset*/
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding:0;
+}
+/*variabelen*/
+
+:root {
+    --primary-color:#5722CD;
+    --primary-color-light:  #4845E4;
+    --default-radius: 3px;
+
+}
+
+/*/////////////////////////////////////////
+        3.Layout
+////////////////////////////////////////// */
+
+.intro__outer-container,
```

Comment:
Dit is een resuable, die hoef je niet per se dan intr, header, project etc te noemen. Deze had je simpelweg outer-container / inner-container kunnen noemen en gebruiken op ieder stuk op je HTML!